### PR TITLE
[terra-layout]Remove Terra.it helpers from wdio

### PR DESCRIPTION
**Pull request by saket2403**
### Summary
Remove Terra.it helpers from wdio as they are deprecated

### Deployment Link

https://terra-framework-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-layout/tests/wdio/layout-slide-panel-spec.js [hunk 1]
```diff
@@ -1,33 +1,30 @@
 /* globals $ */
 Terra.describeViewports('Toggle the slide panel and hidden styles', ['medium'], () => {
-  before(() => {
+  it('toggles the slide panel and hidden styles', () => {
     browser.url('/#/raw/tests/terra-layout/layout/layout-slide-panel-example');
```
Comment (mjhenkes) [inline packages/terra-layout/tests/wdio/layout-slide-panel-spec.js:4]: Remove the '#' please.

File: packages/terra-layout/tests/wdio/layout-slide-panel-spec.js [hunk 2]
```diff
@@ -1,33 +1,30 @@
...
+    Terra.validates.element('toggle slide panel', { selector: '#site' });
   });
-
-  Terra.it.matchesScreenshot({ selector: '#site' });
 });
 
 describe('Toggle the slide panel click', () => {
-  before(() => {
+  it('toggles the panel with click', () => {
     browser.url('/#/raw/tests/terra-layout/layout/layout-slide-panel-example');
```
[the remark this answers is not in the document]
Comment (saket2403) [reply packages/terra-layout/tests/wdio/layout-slide-panel-spec.js:12]: The build fails with a diff of 0.02 on travis only, on removing the `/#`.

[the remark this answers is not in the document]
Comment (mjhenkes) [reply packages/terra-layout/tests/wdio/layout-slide-panel-spec.js:12]: @saket2403 I'd like you to further investigate the cause. It's interesting that removing a redirect is causing a 0.02 diff on travis.

**Review by mjhenkes — CHANGES_REQUESTED** [review]
We should be able to remove the '/#' redirect.

**Comment by saket2403** [comment]
> We should be able to remove the '/#' redirect.

Updated wdio 44a238b. Got the screenshots to fail in local by adding FORM_FACTOR in package.json.

**Review by mjhenkes — APPROVED** [review]

**Review by pranav300 — APPROVED** [review]

**Review by supreethmr — APPROVED** [review]

**Review by ryanthemanuel — APPROVED** [review]

**Pull request merged**